Makefile

```diff
@@ -31,13 +31,13 @@ clean:
 .PHONY: golangci-lint
 golangci-lint:
 	@if $(GOPATH)/bin/golangci-lint version 2>&1 | grep -vq $(GOLANGCI_LINT_VERSION); then\
-		pushd /tmp && GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION) && popd; \
+		pushd /tmp && GO111MODULE=on go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION) && popd; \
 	fi
```

Comment:
This is unrelated to this commit, see https://github.com/code-ready/machine-driver-libvirt/pull/98/commits/cb1739f048f7ad03c39247e2af6e82e052c1f156 for an alternate way of handling this (it's the same code in crc)

Reply:
handled it the same way, thanks